desktop-environment.el

```diff
@@ -41,6 +41,16 @@
   :group 'environment)
 
 
+;;; Customization - keyboard backlight
+(defcustom desktop-environment-keyboard-backlight-normal-increment "1"
+  "Normal keyboard increment value."
+  :type 'string)
```

Comment:
it seems that you are using this variable as a number, not as a string. I would change the type and default value then.

Reply:
Changed to integer.